frontend/DataView/Simple.js

```diff
@@ -200,6 +200,8 @@ function textToValue(txt) {
 function valueToText(value) {
   if (value === undefined) {
     return 'undefined';
+  } else if (value === Infinity) {
```

Comment:
Good idea, and I wouldn't bother too much about `-0` since one can argue that it's exactly equal to `0` :)